x/emissions/proto/emissions/v1/params.proto

```diff
@@ -123,4 +123,10 @@ message Params {
     (gogoproto.nullable) = false
   ];  // we no stop dripping from the topic's effective revenue when the topic's
       // effective revenue is below this
+  string data_sending_fee = 42 [
```

Comment:
You have to add more lines of code to properly add a new global param:
https://www.notion.so/upshothq/Adding-New-Global-Params-6216783e3b4f4280a2fda6f3487f0164

note there's validation to add as well (as pointed out in the doc, just calling your attention to it)